src/user/components/LoginPage.vue

```diff
@@ -50,13 +53,19 @@ export default defineComponent({
   methods: {
     logIn() {
       client.login(this.username, this.password).then(() => {
+        this.$emit("close");
         this.$router.push(this.redirectTo);
       });
     },
     logInViaGoogle() {
       client.googleLogin().then(() => {
+        this.$emit("close");
         this.$router.push(this.redirectTo);
       });
+    },
+    close() {
+      this.$emit("close");
+      this.$router.push(this.redirectTo);
```

Comment:
I think we shouldn't redirect the user if they close the login, only if they actually log in.